roles/vmware-ci-set-passwords/files/vcenter_set_password

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/python3
```

Comment:
Should add LICENSE header

Reply:
Also, there is no way to do this in ansible today?

Reply:
No, not really.